node/windows-packaging/CalicoWindows/kubernetes/kubelet-service.ps1

```diff
@@ -65,7 +65,7 @@ else
     $argList += "--cni-bin-dir=""c:\k\cni"""
     $argList += "--cni-conf-dir=""c:\k\cni\config"""
     $argList += "--network-plugin=cni"
-    $argList += "--pod-infra-container-image=kubeletwin/pause"
+    $argList += "--pod-infra-container-image=mcr.microsoft.com/oss/kubernetes/pause:3.6"
```

Comment:
I suggest we keep using `kubeletwin/pause` in `kubelet-service.ps1`.  We can update our installation script to 
- Download official pause image
- retag it to `kubeletwin/pause`

This approach keeps backward compatibility that pause image name has not been changed and is ready after installation.  It also allows user to install a different pause image if they want.

Reply:
We talked about this. I've made the change. Thinking about it some more I was a bit unsure about having a local image tag. But I think the main benefit to pre-pulling is figuring out during install time if the pause image exists or not.